src/renderer/services/file-management-system/index.ts

```diff
@@ -511,7 +524,11 @@ export default class FileManagementSystem {
     
     // For rate throttling how many chunks are sent in parallel
     let chunksInFlight = 0;
-    const chunksInFlightLimit = FileManagementSystem.CHUNKS_CEILING_INFLIGHT_REQUEST_CEILING;
+
+    const chunksInFlightLimit = FileManagementSystem.chunksInFlightMaxForChunkSize(chunkSize);
+    //TODO remove after we observe bugfix effacacy in the field -chrishu 3/7/23
+    console.log("chunksInFlightLimit: " + chunksInFlightLimit)
+    console.log("chunkSize: " + chunkSize)
```

Comment:
Should we release this as a snapshot from this branch and then remove the log statements before merging this PR?